core/src/worker/base.rs

```diff
@@ -96,6 +96,13 @@ async fn execute_job(
             &msg.message.inputs,
             cfg.openai_api_key.clone(),
         )?,
+        // Adding default model URL for now
+        // Need to decide how the model url will be defined
+        "llama2" => ollama::prepare_ollama_embedding_request(
+            job_meta.clone(), 
+            &msg.message.inputs, 
+            "https://0.0.0.0:11434".to_string()
```

Comment:
Can we read this from a GUC? All these settings are defined in [guc.rs](https://github.com/tembo-io/pg_vectorize/blob/0fa65e92cc7eedee660fdddbf5a6efe62ea780e7/extension/src/guc.rs#L17)

Reply:
Sure!